bigquery_etl/cli/dryrun.py

```diff
@@ -96,19 +97,26 @@ def dryrun(paths, use_cloud_function, validate_schemas, respect_skip, project):
             sys.exit(1)
         client = bigquery.Client(project=project)
 
-    def sql_file_valid(sqlfile):
-        """Dry run the SQL file."""
-        result = DryRun(
-            sqlfile,
-            use_cloud_function=use_cloud_function,
-            client=client,
-            respect_skip=respect_skip,
-        )
-        if validate_schemas:
-            return result.validate_schema()
-        return result.is_valid()
+    sql_file_valid = partial(
+        _sql_file_valid, use_cloud_function, client, respect_skip, validate_schemas
+    )
 
-    with ThreadPool(8) as p:
+    with Pool(8) as p:
```

Comment:
See https://github.com/mozilla/bigquery-etl/pull/2452#discussion_r734678680 for the why's.
~2m vs 9m